### ISSUE TITLE
Improve The UI

### ISSUE THREAD

**Issue by suryapratapsinghsuryavanshi** [suryapratapsinghsuryavanshi/NASA-Media-Search#10]
This issue is for beginners who are interested in contributing to open source, can use your Web Development skill, and improve the UI.

**Comment by Manas146** [comment]
I would like to work on this issue.

**Comment by kanchi12P** [comment]
I would also like to work on this .

**Comment by suryapratapsinghsuryavanshi** [comment]
> I would like to work on this issue.

Yeah @Manas146, make your contribution fast.

**Comment by suryapratapsinghsuryavanshi** [comment]
> I would also like to work on this .

Yeah @kanchi12P, you also make your contribution fast.

**Comment by poorvi-sharma** [comment]
I am new to opensource. I would love to contribute as my first contribution. @suryapratapsinghsuryavanshi

**Comment by suryapratapsinghsuryavanshi** [comment]
> I am new to opensource. I would love to contribute as my first contribution. @suryapratapsinghsuryavanshi

Yes give your contribution we will review it and if it is good we will merge it.

**Comment by thisispriyanshu** [comment]
I want to work on this issue. Please assign it to me.

**Comment by Ekanshipal** [comment]
I would like to work on this issue. Please assign it to me @suryapratapsinghsuryavanshi

**Comment by suryapratapsinghsuryavanshi** [comment]
Thank you for showing your interest in this issue we can't assign this issue to anyone but you can do your work we will definitely merge it.
@thisispriyanshu, @Ekanshipal

**Comment by Yenniferh** [comment]
Hello, I would like to work on this

**Comment by suryapratapsinghsuryavanshi** [comment]
> Hello, I would like to work on this

Yes, you can contribute it. This issue is open for Everyone.
